.github/workflows/ci.yml

```
@@ -226,8 +226,47 @@ jobs:
       env:
         UPLOAD_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  website:
+  update-packages:
+    name: Update packages
     needs: launchers
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: true
+      - uses: actions/download-artifact@v2
+        with:
+          name: launchers
+          path: artifacts/
+      - name: Display structure of downloaded files
+        run: ls -R
+        working-directory: artifacts/
+      - uses: webfactory/ssh-agent@v0.5.3
```

Just in case, let's use full commit numbers rather than tags, like
```suggestion
      - uses: webfactory/ssh-agent@5f066a372ec13036ab7cb9a8adf18c936f8d2043
```

It's safer, securitywise (a tag can be erased, while a commit number can't really).